home/graphql-codegen.yml

```diff
@@ -0,0 +1,26 @@
+overwrite: true
+schema: 'http://localhost:3000/--/graphql'
```

Comment:
```suggestion
schema: 'http://localhost:3000/--/graphql'
```
```suggestion
schema: 'https://exp.host/--/graphql'
```

Reply:
i see this was resolved - curious on the context